Observer/AdyenSalesOrderChargedCurrencyObserver.php

```diff
@@ -47,6 +47,11 @@ public function execute(Observer $observer)
         /** @var \Magento\Sales\Model\Order $order */
         $order = $observer->getEvent()->getOrder();
         $paymentMethod = $order->getPayment()->getMethod();
+
+        if (is_null($paymentMethod)) {
+            return false;
```

Comment:
Return type for `execute` is void
```suggestion
            return;
```

Reply:
Thank you for your comments @peterojo